yorkie/src/main/kotlin/dev/yorkie/util/IndexTree.kt

```diff
@@ -453,12 +456,7 @@ internal abstract class IndexTreeNode<T : IndexTreeNode<T>> {
      * the tree is newly created and the size of the descendants is not calculated.
      */
     fun updateDescendantSize(): Int {
-        if (isRemoved) {
-            size = 0
-            return 0
-        }
-
-        size += children.sumOf { it.updateDescendantSize() }
+        size += children.sumOf { if (it.isRemoved) 0 else it.updateDescendantSize() }
```

Comment:
Even if a node is removed, the sizes of the subtree should be updated. So, it would be good to update them by calling `updateDescedantSize()`.

https://github.com/yorkie-team/yorkie-js-sdk/pull/846/files#diff-375b895a8730db047e6747becbb33f2486aa6e6de544f98b8f170ab7c6cb37f4R161

Reply:
I fixed it to call `updateDescendantSize()` even when a node is removed: [c4d7b25](https://github.com/yorkie-team/yorkie-android-sdk/pull/202/commits/c4d7b257f4b53dff8b2e5c992b49bc3691b2c5ac)